packages/celotool/src/lib/generate_utils.ts

```diff
@@ -112,6 +113,7 @@ const validatorBalance = fetchEnvOrFallback(
 ) // 10,011 CG
 const faucetBalance = fetchEnvOrFallback(envVar.FAUCET_GENESIS_BALANCE, '10011000000000000000000') // 10,011 CG
 const oracleBalance = fetchEnvOrFallback(envVar.ORACLE_GENESIS_BALANCE, '100000000000000000000') // 100 CG
+const votingBotBalance = fetchEnvOrFallback(envVar.VOTING_BOT_BALANCE, '10000000000000000000000') // 10,000 CG
```

Comment:
None of these are reading correctly from the .env file, and I don't know why. When I tested generating a genesis file with a .env file that had non-default values for all of these, they did not carry through.

Reply:
@yerdua What do you mean by they are not reading correctly from the `.env` file? Are you sure you did not have another .env file overriding the one you were changing?